pkg/database-types/mysql.go

```diff
@@ -82,6 +84,20 @@ func (m *mysql) TranslateOpenMetadataConfigToFybrikConfig(tableName string,
 	delete(ret, Username)
 	delete(ret, Password)
 
+	if databaseSchema, ok := config[DatabaseSchema]; ok {
+		delete(ret, DatabaseSchema)
+		ret[Database] = databaseSchema
+	}
+
+	if hostPort, ok := config[HostPort]; ok {
+		delete(ret, HostPort)
+		s := strings.Split(hostPort.(string), ":")
+		ret[Host] = s[0]
+		if port, err := strconv.Atoi(s[1]); err != nil {
+			ret[Port] = port
+		}
+	}
```

Comment:
In the end of the process, from all "standardFields" you live only "Schema" in the "ret", is it correct?

Reply:
correct

Reply:
I have since changed the code.
Unlike before, the map which represents the open-metadata configuration remains unchanged.
Here is an example of the original `config`:
```
{
   "type": "Mysql",
   "scheme": "mysql+pymysql",
   "hostPort": "mysql.fybrik-airbyte-sample:3306",
   "databaseSchema": "fybrik",
   "supportsMetadataExtraction": true,
   "supportsProfiler": true,
   "supportsQueryComment": true
}
```
And here is an example of the transformed configuration:
```
{
   "table": "username",
   "database": "fybrik",
   "scheme": "mysql+pymysql",
   "host": "mysql.fybrik-airbyte-sample",
   "port": 3306
   "other": {
      "supportsMetadataExtraction": true,
      "supportsProfiler": true,
      "supportsQueryComment": true
   }
}
```